drivers/gpu/drm/bridge/cdns-mhdp.c

```diff
@@ -950,19 +973,12 @@ static void mhdp_link_training_init(struct cdns_mhdp_device *mhdp)
 	mhdp->link_state.link_rate = mhdp->link.rate;
 	mhdp->link_state.num_lanes = mhdp->link.num_lanes;
 
-	phy_cfg.dp.link_rate = (mhdp->link.rate / 100);
-	phy_cfg.dp.lanes = (mhdp->link.num_lanes);
 	for (i = 0; i < 4; i++) {
 		mhdp->link_state.voltage[i] = 0;
 		mhdp->link_state.pre[i] = 0;
-		phy_cfg.dp.voltage[i] = 0;
-		phy_cfg.dp.pre[i] = 0;
 	}
-	phy_cfg.dp.ssc = false;
-	phy_cfg.dp.set_lanes = true;
-	phy_cfg.dp.set_rate = true;
-	phy_cfg.dp.set_voltages = true;
-	phy_configure(mhdp->phy,  &phy_cfg);
+	
+	mhdp_phy_configure(mhdp, true, true, true);
```

Comment:
Usually these kind of function parameters are not recommended. It's impossible to look at the above line, and even guess what the parameters mean.

Looking at the code, we only have two cases: 1) set all phy configs with VS/PE always 0, 2) only set VS/PE to new values. Would it make sense to have two separate functions for these?